libs/libc/time/lib_localtime.c

```diff
@@ -2801,6 +2801,11 @@ void tzset(void)
   nxrmutex_unlock(&g_lcl_lock);
 }
 
+void tzreset(void)
```

Comment:
This seems to be a non-standard API (not defined by POSIX). If I recall correctly we have two implementations of localtime, so from an API perspective it should be compatible with each other

Reply:
Where do we have second implementation of local time? As far as I was able to find` localtime_r` is also defined in `lib_gmtimer.c` but that is used only if `CONFIG_LIBC_LOCALTIME` is not set (and it isnt really localtime).

Reply:
Yeah, you are right. The another implementation does not have TZ interface